cmake/zstd.cmake

```
@@ -4,7 +4,7 @@
 option(WITH_ZSTD
     "Build support for compression/decompression with Zstd." TRUE)
 if (WITH_ZSTD)
-    find_package(zstd CONFIG QUIET)
+    find_package(Zstd CONFIG QUIET)
```

```suggestion
    find_package(zstd CONFIG QUIET)
```

The config file is `zstdConfig.cmake`. This is the case-sensitive variant, matching only `zstd`.

---

It is maddening this stuff isn't consistent and enforced in some way.